server/src/main/java/com/linecorp/centraldogma/server/internal/api/ContentServiceV1.java

```diff
@@ -98,6 +99,9 @@
 public class ContentServiceV1 extends AbstractService {
 
     private static final String MIRROR_LOCAL_REPO = "localRepo";
+    private static final Predicate<Change<?>> NOT_INTERNAL_FILES =
```

Comment:
Instead of adding it here, how about adding it to `DefaultMetaRepository` and using it here?
```java
public class DefaultMetaRepository ... {

    public static final String PATH_CREDENTIALS = "/credentials.json";

    public static final String PATH_MIRRORS = "/mirrors.json";

    public static final List<String> metaRepoFiles = ImmutableList.of(PATH_CREDENTIALS, PATH_MIRRORS);

    ...
}

final boolean hasChangesOtherThanMetaRepoFiles =
                    Streams.stream(changes).anyMatch(change -> metaRepoFiles.contains(change.path()));
```

Reply:
* `hasChangesOtherThanMetaRepoFiles` -> `hasInternalFiles`
* No strong reason to use `static final Predicate<Change<?>>`. `(change) -> { ... }` is converted into a singleton lambda object that is created only once by JVM.
  Inline lambda expression might be better for readability.

Reply:
Thanks !!

> hasChangesOtherThanMetaRepoFiles -> hasInternalFiles?

I used `hasChangesOtherThanMetaRepoFiles` because we should check files that are not internal.

> public static final List<String> metaRepoFiles = ImmutableList.of(PATH_CREDENTIALS, PATH_MIRRORS);

Like `Project#internalRepos()`, how about making the field private and exposing a method? Maybe it's ok since the `DefaultMetaRepository` is in the internal package?

https://github.com/line/centraldogma/blob/88ce886c8c6bf336839ae36a7c292cdc97137565/server/src/main/java/com/linecorp/centraldogma/server/storage/project/Project.java#L78-L80

Reply:
Either way is fine to me. 😄